plugins/nanospell/plugin.js

```diff
@@ -937,8 +937,55 @@
 					}
 				}
 			);
+
+			range.shrink(CKEDITOR.SHRINK_TEXT);
 			range.extractContents().appendTo(span);
 			range.insertNode(span);
+
+			// In cases where a spellcheck span must cross a nested inline element,
+			// extractContents will duplicate the tag, altering the HTML structure.
+			// The following checks to see if this has happened and corrects it
+			//
+			// Example:
+			// <p><strong>word missspelli</strong>ngs
+			// After spellcheck span insertion:
+			// <p><strong>word </strong><span class="nanospell-typo"><strong>missspelli</strong>ngs</span>
```

Comment:
is this the case that it's trying to fix?

or is this the actual intended output?  (I suspect former)